model/src/main/java/com/javahelp/model/user/ProviderUserInfo.java

```diff
@@ -98,6 +100,30 @@ public void setPhoneNumber(String phoneNumber) {
         this.phoneNumber = phoneNumber;
     }
 
+    /**
+     * Gets the gender of this provider
+     *
+     * @return the current gender of this client
+     */
+    public String getGender(){
+        return gender;
+    }
+
+    /**
+     * Sets a new gender to this client
+     *
+     * @param gender an updated gender of this client
+     */
+    public void setGender(String gender){
+        if (!(Gender.FEMALE.toString().equals(gender.toUpperCase()) ||
+                Gender.MALE.toString().equals(gender.toUpperCase()) ||
+                Gender.OTHERS.toString().equals(gender.toUpperCase()))){
+            throw new IllegalArgumentException("This is not a valid input. Please try \"MALE\"" +
+                    "or \"FEMALE\" or \"OTHERS\"");
+        }
+        this.gender = gender.toUpperCase();
```

Comment:
Instead of setting and storing gender as a `String`, if you are going to have an enum for it, it probably makes more sense to just set/store it as an enum. So instead of `String gender`, you would have `Gender gender`.